RecoMET/METFilters/python/ecalBadCalibFilter_cfi.py

```diff
@@ -29,4 +29,39 @@
                                                        872423215,872416066,872435036,872439336,
                                                        872420273,872436907,872420147,872439731,
                                                        872436657,872420397,872439732,872439339,
-                                                       872439603,872422436])
+                                                       872439603,872422436,872439861,872437051,
+                                                       872437052,872420649,872422436,872421950,
+                                                       872437185,872422564,872421566,872421695,
+                                                       872421955,872421567,872437184,872421951,
+                                                       872421694,872437056,872437057,872437313])
+
+
+ecalBadCalibReducedMINIAODPre104XFilter = cms.EDFilter(
```

Comment:
The settings should not be copy-pasted from what is in ecalBadCalibFilter.
Presently, ```ecalBadCalibReducedMINIAODPre104XFilter = ecalBadCalibFilter.clone(EcalRecHitSource = 'reducedEgamma:reducedEERecHits')``` should give identical result.

I don't understand the need to have "Pre104X" in the name. IIUC, the simplest is to just apply it to all data anyways on top of the default ecalBadCalibFilter. Either it's an analysis running on miniAOD made with the most up-to-date list of xtals [in this case this filter is redundant or just a subset of the full one ecalBadCalibFilter] or your miniAOD doesn't cover all xtals in which this ecalBadCalibReducedMINIAODFilter triggers (partially/the best way possible)  on the bad events.

Also, nominally, we do not allow multiple modules defined in cfi files.
So, this definition should be moved to a separate _cff file.